src/main/java/frc/robot/constants/VisionConstants.java

```diff
@@ -34,29 +45,29 @@ public static void update(RobotId robotID) {
         new Pair<String, Transform3d>(
           "Camera_2",
           new Transform3d(
-            new Translation3d(-Units.inchesToMeters(4.75), Units.inchesToMeters(10.375), Units.inchesToMeters(10)),
-            new Rotation3d(0, 0, 0)
+            new Translation3d(Units.inchesToMeters(15), Units.inchesToMeters(-3.3125), Units.inchesToMeters(10)),
+            new Rotation3d(0, 0, Math.PI)
           )
         )
       ));
     } else {
-      // kCameras = new ArrayList<Pair<String, Transform3d>>(List.of(
-      //   new Pair<String, Transform3d>(
-      //     "Camera_1",
-      //     new Transform3d(
-      //       new Translation3d(Units.inchesToMeters(1.25), Units.inchesToMeters(7.125), Units.inchesToMeters(21)),
-      //       new Rotation3d(0, 0, 0)
-      //     )
-      //   ),
-
-      //   new Pair<String, Transform3d>(
-      //     "Camera_2",
-      //     new Transform3d(
-      //       new Translation3d(Units.inchesToMeters(1.25), Units.inchesToMeters(-7.125), Units.inchesToMeters(21)),
-      //       new Rotation3d(0, 0, 0)
-      //     )
-      //   )
-      // ));
+      // DO NOT COMMENT THIS OUT
```

Comment:
I think the comment is unnecessary

Reply:
It is necessary. This was commented out during SFR instead of disabling using the variable. I don't want it to be commented out again.

Reply:
![image](https://user-images.githubusercontent.com/113394349/227814971-776940ce-8fe3-4b75-987e-14e95badba1f.png)
This is why it is necessary.

Reply:
There are reasons to remove the cameras.
For example if mechanical removes a camera.
Another example is the sim, which doesn't support the cameras (though there should be some code to support that).
In general, writing a comment like "don't comment this out" is strange because there are cases where you want to disable things. Commenting is a way of writing code, it would be like writing "don't write code". Now there are some cases where maybe some code is very crucial and should not be modified at all (like Main.java, which has a comment), but that's not the case for VisionConstants.java